ini/stm32g0.ini

```diff
@@ -32,7 +32,7 @@ build_flags = -DPIN_WIRE_SCL=PB3 -DPIN_WIRE_SDA=PB4
 [env:BTT_EBB42_V1_1_filament_extruder]
 extends                     = stm32_variant
 platform                    = ststm32@~14.1.0
-platform_packages           = framework-arduinoststm32@https://github.com/stm32duino/Arduino_Core_STM32/archive/main.zip
+platform_packages           = framework-arduinoststm32@https://github.com/stm32duino/Arduino_Core_STM32/archive/6f39b418e53bf827650399feacda9c3d3b245b57.zip
```

Comment:
Since `Arduino_Core_STM32` actually does releases, I think it'd be better to use them instead of commit: https://github.com/stm32duino/Arduino_Core_STM32/releases